deepchecks/utils/distribution/drift.py

```diff
@@ -162,9 +166,12 @@ def psi(dist1: Union[np.ndarray, pd.Series], dist2: Union[np.ndarray, pd.Series]
     psi_value = 0
     for i in range(len(expected_counts)):
         # In order for the value not to diverge, we cap our min percentage value
-        e_perc = max(expected_counts[i] / size_expected, PSI_MIN_PERCENTAGE)
-        a_perc = max(actual_counts[i] / size_actual, PSI_MIN_PERCENTAGE)
-        value = (e_perc - a_perc) * np.log(e_perc / a_perc)
```

Comment:
problematic change here. ``min_category_size_ratio`` is used for the ``preprocess_2_cat_cols_to_same_bins`` thingy. First of all, it means that this parameter is in face meaningless here, as after the preprocessing you only get categories that are > ``min_category_size_ratio``.

But also, you're using the same parameter for 2 different functionalities, even if they are very similar

Reply:
That's what I was thinking... why do we need the PSI_MIN_PERCENTAGE to be different than min_category_size_ratio? I get that we don't want someone to set min_category_size_ratio=0 and get inf values, maybe limit it so that min_category_size_ratio can't be 0?